tests/src/main/java/com/hazelcast/simulator/tests/replicatedmap/ReplicatedMapTest.java

```diff
@@ -91,12 +94,15 @@ public Worker createWorker() {
 
     private class Worker extends AbstractWorker<Operation> {
 
+        private final Metronome metronome = SimpleMetronome.withFixedIntervalMs(intervalMs);
```

Comment:
Should be implemented and used via https://github.com/hazelcast/hazelcast-simulator/commit/348c2cd767f7af906b00af1949de48b6492de8a2